getMultiStarts.m

```diff
@@ -295,7 +318,7 @@
                 % parameters.constraints.A  ,parameters.constraints.b  ,... % linear inequality constraints
                 % parameters.constraints.Aeq,parameters.constraints.beq,... % linear equality constraints
                 
-                parameters.MS.J(1, i) = -J_0;
+                parameters.MS.J(1, i) = nan;
```

Comment:
this loop looks better on the develop_minibatch branch. However, merging it in would mean merging also the minibatch routines and a lot of not so well tested stuff... I'll leave that for (way) later pull-request...